actions/lib/prepare_archive_package.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Simple script that should be executed on a biotank when we want to prepare a
+# runfolder to be archived to PDC via TSM.
+
+set -e
+
+# The full path to the runfolder that we want to archive.
+RUNFOLDER=$1
+
+RUNFOLDER_NAME=$(basename ${RUNFOLDER})
+
+# The exclude pattern is configurable upstreams, but it should hopefully look
+# something like the following. Escaping various quotes can be messy though. 
+# EXCLUDE="^./Config|^./InterOp|^./SampleSheet.csv|^./Unaligned|^./Data|^./Thumbnail_Images"
+EXCLUDE=$2
+
+cd ${RUNFOLDER}
+
+# TODO: Shall we upload it when it exists?
```

Comment:
It could e.g. be a force flag that tries to upload the tar file again. But at the same time it might be argued that we, at least in the beginning, would like to see why these things occur. I.e. it might just be that the upload has failed, and the archive is complete, then it should be OK. But if the archive exists, but e.g. failed to be complete due to full disk space or similar then the following upload would be incomplete/broken.